packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -431,13 +471,10 @@ class _CupertinoTabScaffoldState extends State<CupertinoTabScaffold> {
 
   @override
   void dispose() {
-    // Only dispose `_controller` when the state instance owns it.
-    if (widget.controller == null) {
-      _controller?.dispose();
-    } else if (_controller?._isDisposed == false) {
-      _controller!.removeListener(_onCurrentIndexChange);
+    if (widget.controller?._isDisposed == false) {
+      _controller.removeListener(_onCurrentIndexChange);
     }
-
+   _internalController?.dispose();
```

Comment:
I think the indentation is off here if I'm seeing this right.

Reply:
Good eye!